src/lib/index.js

```
@@ -124,212 +124,212 @@
 k.set("stream", ue);
-k.set("update_display_data", S);
-function M(n) {
-  const e = k.get(n.output_type);
+k.set("update_display_data", M);
+function S(r) {
+  const e = k.get(r.output_type);
   let t;
   return (
     e
-      ? (t = e(n))
+      ? (t = e(r))
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)